.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+end_of_line = 1f
+insert_final_newline = true
+indent_style = space
+
+[*.php]
+indent_size = 4
```

Comment:
That's what I meant, can we just apply this to the `[*]` block immediately under `indent_style`.